src/JupyterLibrary/clients/jupyterlab/Notebook.robot

```diff
@@ -51,3 +51,14 @@ Run Current JupyterLab Code Cell
     ...    ${nb.find_element_by_xpath('''div${JLAB XP NB TOOLBAR FRAG}//${run icon}''')}
     Click Element    ${run btn}
     Sleep    0.5s
+
+Get Selected Tab ID
+    [Documentation]    Return the tab ID of the selected notebook tab
+    ${active-nb-tab} =    Get WebElement    xpath:${JL_TABBAR_SELECTED_XPATH}
+    ${tab-id} =    Get Element Attribute    ${active-nb-tab}    id
```

Comment:
This seems like a useful capability.

For consistency, it should be added in `Shell.robot` (well, now `Shell.resource`) and probably something more like the existing keywords, e.g. `Get JupyterLab Dock Panel Tab` so perhaps `Get Selected JupyterLab Dock Panel Tab` and likely return the actual web element.

This can be a one-shot, as `Get Element Attribute` will accept `xpath:`. Further, it should 

I don't see where `JL_TABBAR_SELECTED_XPATH` is defined, but would expect to conform to the  (increasingly ridiculous, sorry) naming convention.

Reply:
Hi, yeah sorry the JL_TABBAR_SELECTED_XPATH is missing, my bad. I will fix it.

Thanks for the review. I will go through all the comments asap.

Reply:
@bollwyvl I move `Get Selected JupyterLab Dock Panel Tab` in Shell.resource and instead of returning only the tab ID, I return both the web element and the id. In this way i think the keyword is more useful to the user. 
Do you agree with this solution?